web/apps/benchmarks-and-reports/src/app/(benchmarks-and-reports)/benchmarks/_components/charts.tsx

```diff
@@ -143,7 +143,7 @@ export function Charts() {
       <FooterAscii text="Benchmarks" />
 
       <Script
-        src="https://risc0.github.io/ghpages/dev/bench/data.js"
+        src="https://reports.risczero.com/benchmarks"
         onReady={() => {
```

Comment:
this likely was a mistake that broke the vercel job? cc @flaub I should have spotted https://github.com/risc0/ghpages/tree/main/dev/bench/data.js and other scripts are not in the new hosted site I can see. I should not have used this incorrect link either. Sorry! 
![image](https://github.com/risc0/risc0/assets/35669742/b1354f2a-a1b3-49a1-aff8-935fb54df759)

Reply:
Yep, I'm on it